### ISSUE TITLE
Abbreviation "OB" in parameter description

### ISSUE THREAD

**Issue by rartych** [camaraproject/KnowYourCustomer#82]
**Problem description**
In both API definitions the parameters are described as "stored on the **OB** systems"

```
        phoneNumber:
          type: string
          description: The phone number of the customer stored on the OB systems.
```

**Expected behavior**
The abbreviation is proprietary and the expression should be replaced with, for example: "_stored in the Operator's system_"

**Alternative solution**
Better expression can be proposed by native speaker.

**Additional context**

**Comment by fernandopradocabrillo** [comment]
Hi @rartych 
yes I agree, this is a term that we use in Telefónica that has slipped into the definitions from the initial proposal.
Do we have something standard? Your proposal is fine from my side

**Comment by ToshiWakayama-KDDI** [comment]
Hi @rartych . @fernandopradocabrillo ,

Rafal's proposal is fine from my side, too.  Let's have it in the PR #80 correction, shall we?

BR
Toshi

**Comment by fernandopradocabrillo** [comment]
Hi @rartych @ToshiWakayama-KDDI 

Included in PR #80. Thanks!

**Comment by ToshiWakayama-KDDI** [comment]
Hi @fernandopradocabrillo ,

Thank you so much!  Sorry for the late response.

Best regards,
Toshi
